operating/manage-security-policies.adoc

```diff
@@ -18,6 +18,8 @@ include::modules/modify-existing-security-policies.adoc[leveloffset=+1]
 * xref:../operating/manage-security-policies.adoc#create-policy-from-system-policies-view_manage-security-policies[Creating a security policy from the system policies view]
 
 include::modules/create-policy-categories.adoc[leveloffset=+1]
```

Comment:
I would recommend that you make an umbrella heading (like the "Creating custom policies" one below) to house your 3 options. That makes it clearer that you really only need to choose one of the ways.